lrlex/src/lib/parser.rs

```diff
@@ -379,51 +379,85 @@ impl<StorageT: TryFrom<usize>> LexParser<StorageT> {
             ///
             /// Meaning: The character 'c', unchanged.
             fn unescape(re: Cow<str>) -> Cow<str> {
+                // POSIX lex has two layers of escaping, there are escapes for the regular expressions themselves
```

Comment:
I'm a nitpicker, but can we format the comment to a consistent line length?

Reply:
Should be fixed in https://github.com/softdevteam/grmtools/pull/333/commits/819f4e9b626ae3e7f5d631757611c349a0013953